src/kirby/components/modal/services/modal.controller.ts

```diff
@@ -7,11 +7,20 @@ import { ActionSheetHelper } from './action-sheet.helper';
 import { ModalConfig } from '../modal-wrapper/config/modal-config';
 import { ActionSheetConfig } from '../action-sheet/config/action-sheet-config';
 import { AlertConfig } from '../alert/config/alert-config';
+import { KirbyAnimation } from '@kirbydesign/designsystem/animation/kirby-animation';
+
+export const modalScrollNoop = () => {
```

Comment:
Consider naming this differently... since, this is not a no-op implementation of a callback (it **will** throw an error). Maybe `defaultScrollCallback` is more appropriate?

Reply:
I agree